### ISSUE TITLE
how long the training takes with ....

### ISSUE THREAD

**Issue by SeekPoint** [pbhatia243/Neural_Conversation_Models#6]
python neural_conversation_model.py --train_dir ubuntu/ --en_vocab_size 60000 --size 512 --data_path ubuntu/train.tsv --dev_data ubuntu/valid.tsv  --vocab_path ubuntu/60k_vocan.en --attention

*****************************************
right now, it takes 3 days with a GTX1080 8G

Running epochs
global step 372800 learning rate 0.1905 step-time 0.91 perplexity 1.00
  eval: bucket 0 perplexity 15943204.21
  eval: bucket 1 perplexity 12256693.99
  eval: bucket 2 perplexity 4716758.96
  eval: bucket 3 perplexity 46789217.08
Running epochs
global step 373200 learning rate 0.1905 step-time 1.04 perplexity 1.00

**Comment by SeekPoint** [comment]
right now, with 3 days gtx 1080, it reaches:
global step 336000 learning rate 0.2086 step-time 0.37 perplexity 1.00
  eval: bucket 0 perplexity 12841500.01
  eval: bucket 1 perplexity 13475859.35
  eval: bucket 2 perplexity 33048668.01
  eval: bucket 3 perplexity 5675569.78


How can I elegantly exit or How long it takes to done normally?

**Comment by pbhatia243** [comment]
2 - 3 days on GPU
